gui/src/components/Layout.svelte

```diff
@@ -42,6 +43,9 @@
   <div>
     <slot />
   </div>
+  <footer>
```

Comment:
Do we want this in the footer all the time?

We should probably stuff it in to a menu somewhere once we have a menu somewhere to stuff it :)